src/test/java/com/shapesecurity/salvation/PolicyQueryingTest.java

```diff
@@ -109,6 +109,31 @@
         assertFalse("resource is not allowed", p.allowsFrameFromSource(URI.parse("http://www.def.am:555")));
         assertFalse("resource is not allowed", p.allowsChildFromSource(URI.parse("https://www.def.am:555")));
         assertTrue("resource is allowed", p.allowsChildFromSource(URI.parse("http://www.def.am:555")));
+        
+        p = Parser.parse("font-src https://font.com http://font.org", URI.parse("https://abc.com"));
+        assertTrue("resource is allowed", p.allowsFontFromSource(URI.parse("https://font.com")));
+        assertFalse("resource is not allowed", p.allowsFontFromSource(URI.parse("https://font.com:555")));
```

Comment:
This had to be asserted false to pass. However, does that reveal an issue with the library?

Shouldn't the directive `font-src https://font.com http://font.org` match https on any port?